rpc/wrtc_signaling_answerer.go

```diff
@@ -238,6 +245,14 @@ func (ans *webrtcSignalingAnswerer) Stop() {
 // the designated WebRTC data channel is passed off to the underlying Server which
 // is then used as the server end of a gRPC connection.
 func (ans *webrtcSignalingAnswerer) answer(client webrtcpb.SignalingService_AnswerClient) (err error) {
+	// Maintain and eventually log a collection of stats for each answering attempt.
+	stats := &answererStats{}
+	defer func() {
+		if ans.logStats {
+			stats.log(ans.logger)
```

Comment:
This log now looks like:
```
2024-04-29T14:03:46.512-0400	INFO	robot_server.external_signaler	rpc/answerer_stats.go:72	Connection establishment succeeded	{"answerRequestInitReceieved": "2024-04-29T14:03:46.280-0400", "numAnswerUpdates": 4, "averageAnswerUpdate": "172.614µs", "maxAnswerUpdate": "431.875µs", "localICECandidates": ["at: 2024-04-29T14:03:46.287-0400, candidate: udp4 host 127.0.0.1:62814", "at: 2024-04-29T14:03:46.287-0400, candidate: udp4 host 192.168.10.28:58479", "at: 2024-04-29T14:03:46.287-0400, candidate: udp4 host 192.168.64.1:55785", "at: 2024-04-29T14:03:46.343-0400, candidate: udp4 srflx 24.61.184.231:59442 related 0.0.0.0:59442"], "remoteICECandidates": ["at: 2024-04-29T14:03:46.449-0400, candidate: candidate:2894675966 1 udp 1694498815 24.61.184.231 55170 typ srflx raddr 0.0.0.0 rport 55170", "at: 2024-04-29T14:03:46.449-0400, candidate: candidate:2894675966 1 udp 1694498815 24.61.184.231 52601 typ srflx raddr 0.0.0.0 rport 52601", "at: 2024-04-29T14:03:46.449-0400, candidate: candidate:1308079556 1 udp 2130706431 6e5bce95-ff44-4118-a1a5-34ea9384146b.local 50546 typ host", "at: 2024-04-29T14:03:46.449-0400, candidate: candidate:1308079556 1 udp 2130706431 6e5bce95-ff44-4118-a1a5-34ea9384146b.local 52462 typ host", "at: 2024-04-29T14:03:46.449-0400, candidate: candidate:1196105838 1 udp 16777215 34.203.251.176 27900 typ relay raddr 0.0.0.0 rport 62505", "at: 2024-04-29T14:03:46.456-0400, candidate: candidate:1308079556 1 udp 2130706431 6e5bce95-ff44-4118-a1a5-34ea9384146b.local 61301 typ host"]}
```

Those measurements of answer update timings are very short; that's because sending an `AnswerResponse_Update` back to the signaling server does not include the time taken to actually update the database; we're using a bidi stream here, so the sending of a message does not include waiting for any response from the other side.

Reply:
would it make sense to try/is it possible to measure the time to update the database? related question is, how useful is the number if it only captures the time to send the request and nothing after?

Reply:
> to measure the time to update the database

From the answerer side, I don't think this is possible given the current setup. I think the timing we care about is a combination of timings from the answerer and the client:
- When did the answerer generate the candidate?
- When did the answerer send said candidate to the signaling server?
- When did the client receive said candidate from the signaling server?

Those would be understood by looking at the timestamps of an SDK and the RDK it's connecting to.

> how useful is the number if it only captures the time to send the request and nothing after?

Yeah good question: I think not that useful. I'll remove them for now.